Automattic/pocket-casts-android
# Migrate episode view holders from data binding

## Description


Related to: #1908 

This PR migrates episode container from data binding to view binding.

## Review hints

1. I've prepared this PR in a way that it's easy to review the change commit by commit, and I advise doing so.
2. I find it helpful to check "Hide white spaces" option. It makes much more easy to review some XML changes, as indentation in the file was changed. 

![image](https://github.com/Automattic/pocket-casts-android/assets/5845095/f778447f-2a75-4c80-81c6-e2bd1695b88c)

## Testing Instructions
Open the list of episodes and make sure it's rendered as expected. Star some podcasts, add bookmarks to them and add/remove from up next queue. Do this for user episodes as well (`Profile` > `Files` list).


## Checklist
 
#### I have tested any UI changes...
